client/lib/media/store.js

```diff
@@ -62,6 +67,14 @@ function receivePage( siteId, items ) {
 	} );
 }
 
+MediaStore.getDirtyImage = () => {
```

Comment:
How do we get it otherwise? By looping through all the images in a post/page and detecting whether it is dirty or not?